rofeeah_ayeni/alpha_challenges/conditionals.md

```diff
@@ -6,16 +6,34 @@ Chartreuse is my favorite color. Magenta is my second favorite. Periwinkle is my
 * In Ruby, write a conditional that will check a variable set to a color and tell me how I feel about it.
 
 ```ruby
-color = "Magenta"
+color = ""
 
-# write your code here
+case color
```

Comment:
What happens if color is "magenta" or "MAGENTA" and not "Magenta"? Can you find a way to handle casing here?